pywinauto/__init__.py

```diff
@@ -163,3 +163,8 @@ def top_from_point(self, x, y):
         """Get wrapper object for top level element at specified screen coordinates (x, y)"""
         top_element_info = self.backend.element_info_class.top_from_point(x, y)
         return self.backend.generic_wrapper_class(top_element_info)
+
+    def from_focused(self):
```

Comment:
Why don't name it `get_active(self)`? It will be consistent across all levels of class hierarchy.

Reply:
done